app/assets/javascripts/responsive-tables.js

```diff
@@ -0,0 +1,68 @@
+$(document).ready(function() {
+  var switched = false;
+  var updateTables = function() {
+    if (($(window).width() < 767) && !switched ){
+      switched = true;
+      $('table.responsive').each(function(i, element) {
+        splitTable($(element));
+      });
+      return true;
+    }
+    else if (switched && ($(window).width() > 767)) {
+      switched = false;
+      $('table.responsive').each(function(i, element) {
+        unsplitTable($(element));
+      });
+    }
+  };
+
+  $(window).load(updateTables);
+  $(window).on('redraw',function(){switched=false;updateTables();});
+  $(window).on('resize', updateTables);
+
+
+	function splitTable(original)
+	{
+		original.wrap('<div class='table-wrapper' />');
```

Comment:
Expected ')' and instead saw 'table'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected an identifier and instead saw ')'.<br>'wrapper' is not defined.